lua/esqueleto/utils.lua

```diff
@@ -47,9 +47,10 @@ local any = function (f, t)
 end
 
 -- Determine if a file matches a gitignore glob pattern
--- TODO
+-- by converting to regex, then vim.regex API.
 local match_gitignore = function (filepath, gitignore_pattern)
```

Comment:
`match_gitignore` still not working as expected since glob patterns, e.g. `.DS_Store` will be automatically appended `^` -> `^.DS_Store$`, which is undesired